mmv1/products/bigqueryanalyticshub/api.yaml

```diff
@@ -92,3 +92,111 @@ objects:
         name: icon
         description: |-
           Base64 encoded image representing the data exchange.
+  - !ruby/object:Api::Resource
+    name: "Listing"
+    min_version: beta
+    base_url: projects/{{project}}/locations/{{location}}/dataExchanges/{{data_exchange_id}}/listings
+    create_url: projects/{{project}}/locations/{{location}}/dataExchanges/{{data_exchange_id}}/listings?listing_id={{listing_id}}
+    self_link: projects/{{project}}/locations/{{location}}/dataExchanges/{{data_exchange_id}}/listings/{{listing_id}}
+    update_verb: :PATCH
+    update_mask: true
```

Comment:
Can you add a test that uses the update functionality?

Reply:
will do

Reply:
done